src/workspace-data/import-jobs.ts

```diff
@@ -27,29 +27,26 @@ export const useImportJobs = (workspace: WorkspaceWrapper): UseImportJobsResult
       workspace: { namespace, name },
     } = workspace;
     try {
-      const runningJobsInWorkspace: { jobId: string }[] = await Ajax(signal)
-        .Workspaces.workspace(namespace, name)
-        .listImportJobs(true);
-
-      asyncImportJobStore.update((previousState) => {
-        return [
-          ...previousState.filter((job) => !isJobInWorkspace(job, workspace)),
-          ...runningJobsInWorkspace.map(({ jobId }) => ({ jobId, targetWorkspace: { namespace, name } })),
-        ];
-      });
+      // Imports into Azure workspaces do not show up in this call to orch since there is no centralized knowledge of azure import jobs
+      // Azure workspaces must rely solely on the asyncImportJobStore to know what imports are currently running,
+      // Therefore they do not need a callback function here
```

Comment:
I don't think this comment really makes sense... `useImportJobs` don't need a centralized list of Azure import jobs because it's given a workspace, from which it can get a WDS URL. So this _could_ still refresh jobs for the given workspace, even though it can't refresh jobs for all workspaces like the GCP version.